packages/typography/src/Link/Link.spec.tsx

```diff
@@ -148,11 +156,85 @@ describe('packages/typography', () => {
     test('it renders the correct tag to the DOM', () => {
       renderLink({
         href: 'http://localhost:9001',
+        /* @ts-expect-error to show that "as" overrides "href" */
         as: 'div',
       });
       const div = screen.getByText('Link').parentElement;
       expect(div).toBeVisible();
       expect(div!.tagName.toLowerCase()).toBe('div');
     });
   });
+
+  describe('anchor props', () => {
+    test('href is always passed to rendered component when the prop is supplied', () => {
+      renderLink({ href: 'test' });
+      const a = screen.getByText('Link').parentElement;
+      expect(a).toHaveAttribute('href');
+      expect(a?.getAttribute('href')).toBe('test');
+    });
+
+    test('it respects "target" and "rel" values when target is set', () => {
```

Comment:
Can we group these `respects "target"` tests in a `describe`?

Reply:
reworked those tests a little bit, lmk what you think